code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -431,6 +431,19 @@
 		/obj/item/weapon/melee/sabre
 		)
 
+
+
+/obj/item/weapon/storage/belt/sabre/AltClick(mob/user)
+	if(contents.len)

```

Comment:
Cant really help today until about nine hours from now. I shall try and see what the issue is then.